src/components/core/geoChart/legends/test/PushpinCategoryLegend.spec.tsx

```diff
@@ -49,4 +52,24 @@ describe("PushpinCategoryLegend", () => {
                 .prop("style"),
         ).toEqual({ color: "#CCCCCC" });
     });
+
+    it("should render FluidLegend component", () => {
+        const wrapper = createComponent({
+            responsive: true,
+            showFluidLegend: true,
+        });
+        const fluidLegend = wrapper.find(FluidLegend);
+        expect(
+            fluidLegend
+                .find(".series .series-name")
+                .first()
+                .prop("style"),
+        ).toEqual({ color: "#6D7680" });
+        expect(
+            fluidLegend
+                .find(".series .series-name")
+                .last()
+                .prop("style"),
+        ).toEqual({ color: "#CCCCCC" });
```

Comment:
should we assert paging button?

Reply:
No we shouldn’t. It is already tested in FluidLegend component